docker/Dockerfile

```diff
@@ -20,7 +21,7 @@ COPY ./videojs-dependencies.yml ./videojs-dependencies.yml
 
 RUN crystal spec --warnings all \
     --link-flags "-lxml2 -llzma"
-
+    
```

Comment:
Can you remove these useless spaces on this line here? There should be no differences here.

Reply:
> Can you remove these useless spaces on this line here? There should be no differences here.

done